### ISSUE TITLE
The service "GET /product/{product}/photo" return a null photo content

### ISSUE THREAD

**Issue by benjaminpochat** [benjaminpochat/localeat#27]
With curl :

```
benjamin@MacBookPro:~/Bureau$ curl -v http://localhost:8080/products/103/photo
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8080 (#0)
> GET /products/103/photo HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.52.1
> Accept: */*
> 
< HTTP/1.1 200 
< Vary: Origin
< Vary: Access-Control-Request-Method
< Vary: Access-Control-Request-Headers
< X-Content-Type-Options: nosniff
< X-XSS-Protection: 1; mode=block
< Cache-Control: no-cache, no-store, max-age=0, must-revalidate
< Pragma: no-cache
< Expires: 0
< X-Frame-Options: DENY
< Content-Type: application/json
< Transfer-Encoding: chunked
< Date: Tue, 09 Feb 2021 20:46:49 GMT
< 
* Curl_http_done: called premature == 0
* Connection #0 to host localhost left intact
null

```
Observed only if the logging level logging.level.org.springframework.web is not DEBUG !
As a workaround, the property logging.level.org.springframework.web is set to DEBUG in application-prod.properties file.
But logging level should not be so high in production.

**Comment by benjaminpochat** [comment]
Reminder :
* test if the problem comes from WebMvcConfig (to align json mapper with jpa annotations)
* submit problem on StackOverFlow

**Comment by benjaminpochat** [comment]
https://stackoverflow.com/questions/66145050/why-spring-rest-controller-with-hibernate5module-return-null-when-logging-leve

**Comment by benjaminpochat** [comment]
fixed with commit 322cfb1881c5ee596bc47faa914b599f9adebf81
